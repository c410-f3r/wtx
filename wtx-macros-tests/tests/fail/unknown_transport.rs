#[wtx_macros::pkg(data_format(json), id(FooId), transport(dasdasd))]
mod pkg {
  #[pkg::req_data]
  struct Req;

  #[pkg::res_data]
  struct Res;
}

fn main() {
}
