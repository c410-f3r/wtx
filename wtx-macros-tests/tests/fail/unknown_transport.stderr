error: Unknown transport.
 --> tests/fail/unknown_transport.rs:1:59
  |
1 | #[wtx_macros::pkg(data_format(json), id(FooId), transport(dasdasd))]
  |                                                           ^^^^^^^

error: aborting due to 1 previous error

